vars/karateFlow.groovy

```diff
@@ -31,18 +31,36 @@ def call(params) {
     }
 
     stage('Run karate tests') {
-        script {
-            def karateEnvironment = "folio-testing-karate"
-            withMaven(jdk: 'openjdk-17-jenkins-slave-all',
-                maven: 'maven3-jenkins-slave-all',
-                mavenSettingsConfig: 'folioci-maven-settings') {
-                def modules = ""
-                if (params.modules) {
-                    modules = "-pl common,testrail-integration," + params.modules
+        steps{
+            script {
+                def karateEnvironment = "folio-testing-karate"
```

Comment:
Dear Renat
Can we have Dynamically karateEnvironment variable's value here? 
f.e something like this style
    parameters {
        string(name: 'KARATE_ENVIRONMENT', defaultValue: 'folio-testing-karate')
    }

    stages {
        stage('Run Karate Tests') {
            steps {
                script {
                    def karateEnvironment = "${params.KARATE_ENVIRONMENT}"

for future if we need to use same job for another env i guess it would be nice to have dynamically
from jenkins job send value 
if will not be used and its only for one env  then ignore my comment pls
thanks

Reply:
Hi Guram, we used tests only for one env.